test/controllers/graphql_controller_5_test.rb

```diff
@@ -405,6 +405,25 @@ def setup
     assert_equal 'Test', JSON.parse(@response.body)['data']['createTag']['tag_text_object']['text']
   end
 
+  test "should get relevant articles for ProjectMedia item" do
+    t = create_team
+    pm = create_project_media quote: 'Foo Bar', team: t
+    ex = create_explainer language: 'en', team: t, title: 'Foo Bar'
+    pm.explainers << ex
+    cd = create_claim_description description: pm.title, project_media: pm
+    fc = create_fact_check claim_description: cd, title: pm.title
+    items = FactCheck.where(id: fc.id) + Explainer.where(id: ex.id)
+    ProjectMedia.any_instance.stubs(:get_similar_articles).returns(items)
+    query = "query { project_media(ids: \"#{pm.id}\") { relevant_articles_count, relevant_articles {  edges { node { ... on FactCheck { dbid }, ... on Explainer { dbid } } } } } }"
```

Comment:
@danielevalverde does this test align with the query used in the front end to pull the relevant_articles?

Reply:
@skyemeedan I added this test as a reference for front-end and I'll add a comment to jira ticket and if there is a missing fields required by @danielevalverde I can add these fields to epic branch